<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CleaningDialog</class>
 <widget class="QDialog" name="CleaningDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>336</width>
     <height>55</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="labelIn">
        <property name="text">
         <string>Wash Input</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelIn_x">
        <property name="text">
         <string>x</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditIn_x"/>
      </item>
      <item>
       <widget class="QLabel" name="labelIn_y">
        <property name="text">
         <string>y</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditIn_y"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>342</width>
     <height>55</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="labelOut">
        <property name="text">
         <string>Waste</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelOut_x">
        <property name="text">
         <string>x</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditOut_x"/>
      </item>
      <item>
       <widget class="QLabel" name="labelOut_y">
        <property name="text">
         <string>y</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditOut_y"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButtonLimit">
      <property name="text">
       <string>Capacity limit on</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonOK">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
